### PR TITLE
Confirm account

### PR DESCRIPTION
**Pull request by rodrieiz**
https://trello.com/c/gyax34HS/16-3-as-a-user-i-should-be-able-to-confirm-my-account-after-signing-up-in-order-to-ensure-that-all-the-users-email-are-valid

### CODE REVIEW

File: app/controllers/confirmations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class ConfirmationsController < DeviseTokenAuth::ConfirmationsController
+  def show
+    render json: {
+      status: 200,
+      success: true,
+      message: 'Your account has been confirmed'
```
Comment (enzofab91) [inline app/controllers/confirmations_controller.rb]: This message could be added with I18n.

File: spec/requests/registration_spec.rb [hunk 2]
```diff
@@ -10,7 +10,8 @@
           gender: 'Female',
           password: 'abcd1234',
           password_confirmation: 'abcd1234'
-        }
+        },
+        confirm_success_url: '/'
```
Comment (enzofab91) [inline spec/requests/registration_spec.rb:16]: What about adding a test to validate that the email es confirmed if token is valid?

**Review by enzofab91 — COMMENTED** [review]
Good job. Left some comments.

File: spec/requests/registration_spec.rb (hunk 2 above)
Comment (rodrieiz) [reply spec/requests/registration_spec.rb:16]: I tried to do it but the test isn't work well. I don't know why but In the browser and also in postman the endpoint works and  I checked the user atributte confimed_at and was updated correctly too.

File: Gemfile [hunk 3]
```diff
@@ -33,6 +33,8 @@ gem 'rubocop-rootstrap'
 gem 'devise'
 gem 'devise_token_auth'
 
+gem 'rails-i18n'
```
Comment (enzofab91) [inline Gemfile]: I think the gem isn't necessary. Have you try it without it?

File: spec/requests/confirmation_spec.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe 'GET registration/confirmation', type: :request do
+  let!(:user) { create(:user, confirmed_at: nil) }
+  subject(:confirmation) { get user_confirmation_path, params: params, as: :json }
```
Comment (enzofab91) [inline spec/requests/confirmation_spec.rb]: You can use subject without a name, if you find it easier you can leave it as you have done it.

```suggestion
  subject { get user_confirmation_path, params: params, as: :json }
```

**Review by enzofab91 — APPROVED** [review]
Good work!

File: config/locales/devise_token_auth.en.yml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+        request_reset_link_msg: "Someone has requested a link to change your password. You can do this through the link below."
+        password_change_link: "Change my password"
+        ignore_mail_msg: "If you didn't request this, please ignore this email."
+        no_changes_msg: "Your password won't change until you access the link above and create a new one."
+      unlock_instructions:
+        account_lock_msg: "Your account has been locked due to an excessive number of unsuccessful sign in attempts."
+        unlock_link_msg: "Click the link below to unlock your account:"
+        unlock_link: "Unlock my account"
+  hello: "hello"
+  welcome: "welcome"
```
Comment (DJA89) [inline config/locales/devise_token_auth.en.yml]: There are some EOL (End of line) missing in a couple of files, you can configure your text editor so they are added automatically on save.

**Review by DJA89 — APPROVED** [review]

File: Gemfile (hunk 3 above)
Comment (rodrieiz) [reply Gemfile]: Yes! It's unnecessary. Thanks

File: config/locales/devise_token_auth.en.yml (hunk 5 above)
Comment (rodrieiz) [reply config/locales/devise_token_auth.en.yml]: Rare, I had done this config before.

**Pull request merged**